trulens_eval/examples/quickstart/snowpark_text2text_quickstart.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Should include some info about snowpark here and why people care.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I'm generally unclear on what the purpose of this notebook is. To show cortex LLM? Logging to snowflake table? Why not cortex feedback provider, cortex search ,etc?